File: VB/FetchAppointmentExample/AppointmentTableAdapterExtended.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace FetchAppointmentExample.ScheduleTestDataSetTableAdapters
    #Region "#AppointmentsTableAdapterEx"
    Partial Public Class AppointmentsTableAdapter
        Public ReadOnly Property Commands() As System.Data.SqlClient.SqlCommand()
            Get
                Return Me._commandCollection
            End Get
        End Property
    End Class
    #End Region ' #AppointmentsTableAdapterEx
End Namespace

File: CS/FetchAppointmentExample/AppointmentTableAdapterExtended.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace FetchAppointmentExample.ScheduleTestDataSetTableAdapters {
    #region #AppointmentsTableAdapterEx
    public partial class AppointmentsTableAdapter {
        public System.Data.SqlClient.SqlCommand[] Commands
        {
            get
            {
                return this._commandCollection;
            }
        }
    }
    #endregion #AppointmentsTableAdapterEx
}
